src/typescript/frontend/src/lib/queries/get-apt-price.ts

```diff
@@ -0,0 +1,31 @@
+"use server";
+
+import { COINGECKO_API_KEY } from "lib/server-env";
```

Comment:
- [ ] Document this in example.env as I said here: https://github.com/econia-labs/emojicoin-dot-fun/pull/492#discussion_r1901981740